package.json

```diff
@@ -21,17 +21,21 @@
     "lint-sol-fix": "prettier --write \"contracts/**/*.sol\""
   },
   "dependencies": {
-    "@openzeppelin/contracts": "5.0.1",
-    "@openzeppelin/contracts-upgradeable": "5.0.1",
-    "@solarity/solidity-lib": "2.6.15"
+    "@iden3/contracts": "1.0.0",
+    "@openzeppelin/contracts": "4.9.5",
+    "@openzeppelin/contracts-upgradeable": "4.9.5",
+    "@rarimo/evm-bridge-contracts": "1.0.0",
```

Comment:
Update this to the latest version please

Reply:
Done